checkstyle/import-control.xml

```diff
@@ -408,6 +408,7 @@
         <allow pkg="com.fasterxml.jackson" />
         <allow pkg="kafka.utils" />
         <allow pkg="org.apache.zookeeper" />
+        <allow pkg="org.apache.log4j" />
```

Comment:
I needed to add this to enable https://github.com/apache/kafka/pull/15044/files#r1430656200